internal/action/actions.go

```diff
@@ -1797,10 +1797,10 @@ func (h *BufPane) PreviousTab() bool {
 	if tabsLen == 1 {
```

Comment:
Hmm, this check is actually not needed anymore either? if `tabsLen` is zero, `Tabs.Active()` can be only zero.

And actually in `NextTab()` similarly.

And in most (all?) other functions too?

Reply:
You are right. Really good catch! Thanks.